index.html

```diff
@@ -0,0 +1,464 @@
+
+    <article class="design flex">
+        <div>
+            <p class="book-title">Featured smashing Book</p>
+            <h1><a href="">Design Systems</a></h1>
+            <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, 
+                making it over 2000 years old.</p>
+            <button class="content">Jump to table contents →</button>
+        </div>
+        <img src="assets/images/cartoon.svg" width="200" height="300" alt="coffe">
+
+    </article>
+
+    <article class="smashing-guides">
+        <article class="smashing-guide">
+          <a href="#"> <h1>JavaScript & <br>Frameworks</h1></a>
+          <p class="smashing-guide1-par">React, Vue, or perhaps the lightweight Alpine.js? Whether you are an experienced JavaScript developer working with one of these frameworks, or just starting to learn, 
+              our authors have created some tutorials for you
+            </p>
+            <button class="btn1">Explore JavaScript & Frameworks</button>
+
+        </article>
+        <article class="smashing-guide">
+            <a href="#"> <h1>JavaScript & <br>Frameworks</h1></a>
+            <p class="smashing-guide1-par">React, Vue, or perhaps the lightweight Alpine.js? Whether you are an experienced JavaScript developer working with one of these frameworks, or just starting to learn, 
+                our authors have created some tutorials for you
+              </p>
+              <button class="btn1">Explore JavaScript & Frameworks</button>
+  
+          </article>
+          <article class="smashing-guide">
+            <a href="#"> <h1>JavaScript & <br>Frameworks</h1></a>
+            <p class="smashing-guide1-par">React, Vue, or perhaps the lightweight Alpine.js? Whether you are an experienced JavaScript developer working with one of these frameworks, or just starting to learn, 
+                our authors have created some tutorials for you
+              </p>
+              <button class="btn1">Explore JavaScript & Frameworks</button>
+  
+          </article>
+          <article class="smashing-guide">
+            <a href="#"> <h1>JavaScript & <br>Frameworks</h1></a>
+            <p class="smashing-guide1-par">React, Vue, or perhaps the lightweight Alpine.js? Whether you are an experienced JavaScript developer working with one of these frameworks, or just starting to learn, 
+                our authors have created some tutorials for you
+              </p>
+              <button class="btn1">Explore JavaScript & Frameworks</button>
+  
+        </article>
+    </article>
+    <div class="guides">
+        <p>SEE ALL GUIDES →</p>
+    </div>
+    <footer>
+        <p class="footer-title">BROWSE ALL SMASHING MAGAZINE TOPICS</p>
+        <div class="footer-link">
+            <a href="#">
+                Acceessibility
+            </a>
+            <a href="#">
+                Acceessibility
+            </a>
```

Comment:
- only break into new lines when the line surpasses the 80 characters. In this case is better to have this formatting:
``` 
<a href="#">Acceessibility</a>
```